Probesys/bileto
# imp: Track emails Message-ID

## Related issue(s)



https://github.com/Probesys/bileto/issues/36

## Changes



- remove the symfony/notifier component as we no longer use it
- move the email notification in a dedicated Message Handler
- add a `Message->emailId` to remember the email `Message-ID` header
- search for `In-Reply-To` when creating tickets from email in order to detect the ticket

## How to test manually



- answer to a ticket
- reply to the notification email and make sure to remove the ID from the subject
- check that the email answers to the ticket

## Checklist
